packages/config/src/options.ts

```diff
@@ -152,6 +152,14 @@ const driverConfigOptions: Array<DriverConfigOption> = [
     validation: validate.isStringOrArrayOfStrings,
     overrideLevel: 'any',
     requireRestartOnChange: 'server',
+  }, {
+    // burnIn config is only considered in run-mode while recording to cypress-cloud
+    name: 'burnIn',
```

Comment:
Shouldn't this be `experimentalBunIn` for now?

Reply:
is the goal to initially release this under an experimental flag?

Reply:
I believe so, here's the relevant doc in ClickUp: https://app.clickup.com/18033298/docs/h6amj-44600/h6amj-58751
@ryanpei can set me straight

Reply:
Got it. `burnIn` was cut over to `experimentalBurnIn` in [6f563ca](https://github.com/cypress-io/cypress/pull/27377/commits/6f563cafcce0f37333d26b86fb4a2054c9fd4da2). should be ready for a re review